tools/image-registry.md

```diff
@@ -2,152 +2,155 @@
+[Kontena Cloud](https://kontena.io/cloud) Image Registry provides a zero-maintenance, secure and ready-to-go solution for storing Docker images.
 
-Kontena may be used with any Docker Image Registry. Users looking for a zero-maintenance, ready-to-go solution are encouraged to consider [Docker Hub](https://hub.docker.com/account/signup/) or [Quay](https://quay.io/). Both of these services provide a hosted registry, as well as some advanced features.
+## Using Kontena Cloud Image Registry
 
-You should use Kontena's built-in Image Registry if you want to:
+* [Prerequisites](./image-registry.md#prerequisites)
+* [Using with Docker CLI](./image-registry.md#using-with-docker-cli)
+  * [Login](./image-registry.md#login-with-docker-cli)
+  * [Build & push](./image-registry.md#build-push-an-image-with-docker-cli)
+  * [Pull](./image-registry.md#pull-an-image-with-docker-cli)
+* [List repositories](./image-registry#list-repositories)
+* [Create a repository](./image-registry#create-a-repository)
+* [Show a repository details](./image-registry#show-a-repository-details)
+* [Remove a repository](./image-registry#remove-a-repository)
+* [List a repository tags](./image-registry#create-a-repository-tags)
+* [Using with Kontena Platform](./image-registry#using-repositories-with-kontena-platform)
+  * [Using with Kontena Stacks](./image-registry#using-with-kontena-stacks)
+  * [Add image registry configuration](./image-registry#add-image-registry-configuration)
+  * [List image registry configurations](./image-registry#list-image-registry-configurations)
+  * [Remove an image registry configuration](./image-registry#remove-an-image-registry-configuration)
 
-* Have total control over where your images are being stored
-* Fully own your images' distribution pipeline
-* Ensure access control and security for your own Docker images
+### Prerequisites
 
-## Using Image Registry
+* [Kontena Cloud](https://cloud.kontena.io) account
+* [Kontena CLI](/tools/cli.md) with the `cloud` plugin. If you don't have the plugin installed, you can install (or upgrade) it with the: `kontena plugin install cloud` command.
 
-* [Create Image Registry Service](image-registry.md#create-image-registry-service)
-  * [Local Storage Backend](image-registry.md#local-storage-backend)
-  * [Amazon S3 Storage Backend](image-registry.md#amazon-s3-storage-backend)
-  * [Azure Storage Backend](image-registry.md#azure-storage-backend)
-* [Accessing Image Registry](image-registry.md#accessing-image-registry)
-* [TLS/SSL](image-registry.md#tlsssl)
-* [Authentication](image-registry.md#authentication)
+### Using with Docker CLI
 
-### Create Image Registry Service
+#### Login with Docker CLI
```

Comment:
Placeholder? Or is this actually same as the "Manual way" bit later on?

Reply:
It should be a header for manual/automated way.. but those sub-headers are confusingly same size. Fixed.